docs/tutorials/mastering.md

```diff
@@ -380,7 +1324,7 @@ juju integrate gnbsim:fiveg_gnb_identity nms:fiveg_gnb_identity
 The next steps are performed using a web browser to navigate to the NMS at http://control-plane-nms.control-plane.mgmt/
 
 ```{note}
-The computer that is running the web browser must also use the same DNS server as the rest of the environment.
+The computer that is running the web browser must also use the same DNS server as the rest of the environment !!
```

Comment:
We could probably just use `.nip.io` here like we did for the Getting Started.

Reply:
OK

Reply:
Done